lib/model/model.go

```diff
@@ -2709,6 +2712,20 @@ func (m *syncMutexMap) Get(key string) sync.Mutex {
 // whitespace is collapsed to a single space. Additionally, whitespace at
 // either end is removed.
 func sanitizePath(path string) string {
-	invalid := regexp.MustCompile(`([[:cntrl:]]|[<>:"'/\\|?*\n\r\t \[\]\{\};:!@$%&^#])+`)
-	return strings.TrimSpace(invalid.ReplaceAllString(path, " "))
+	buf := bytes.NewBuffer(make([]byte, 0, len(path)))
```

Comment:
I think this might be slightly more idiomatic/efficient as a strings.Builder?

Reply:
Good point. See updated PR.